build/gulp/scaffold/rollup.config.js

```diff
@@ -0,0 +1,110 @@
+import replace from 'rollup-plugin-replace'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+
+const lodashExports = [
```

Comment:
```
(!) Missing exports
https://rollupjs.org/guide/en#error-name-is-not-exported-by-module-
node_modules/@stardust-ui/react/dist/es/components/Dropdown/Dropdown.js
pickBy is not exported by node_modules/lodash/lodash.js
```

How this is going to scale? Whenever a new function is used, do I need to remember I have to add it here?

Reply:
Yes, it's a real problem 👎  `lodash` is CJS module, so the there is only one option.
If we will use Babel and `babel-plugin-lodash` this problem will gone.

Reply:
Resolved with `external` and `output.globals`, now is much cleaner